api-model/src/converter.rs

```diff
@@ -274,3 +280,33 @@ impl CertificateConfig {
         }
     }
 }
+
+/// Describes the information required to access CCM
+#[derive(Clone, Eq, PartialEq, Debug)]
+#[cfg_attr(feature = "serde", derive(Serialize, Deserialize))]
+pub struct CcmConfiguration {
+    pub ccm_url: String,
+}
+
+impl Default for CcmConfiguration {
+    fn default() -> Self {
+        CcmConfiguration {
+            ccm_url: "ccm.fortanix.com:443".to_string(),
+        }
+    }
+}
+
+/// Describes the information required to access DSM
+#[derive(Clone, Eq, PartialEq, Debug)]
+#[cfg_attr(feature = "serde", derive(Serialize, Deserialize))]
+pub struct DsmConfiguration {
+    pub dsm_url: String,
```

Comment:
I was expecting a `Url` here, not a `String`. I guess it's too late now as it passed API review?

Reply:
Yes, but I do check at conversion time that it can be parsed as a Url else I throw an error.

Reply:
Also we pass this parameter as a string to sdkms client so I think it's okay to leave it as a string.